api-tests/management/alerting/alerting_test.go

```diff
@@ -16,121 +16,46 @@
 package alerting
 
 import (
+	"context"
+	"encoding/base64"
 	"fmt"
 	"os"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/AlekSi/pointer"
 	"github.com/google/uuid"
 	"github.com/percona-platform/saas/pkg/alert"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
 	"gopkg.in/yaml.v3"
 
 	pmmapitests "github.com/percona/pmm/api-tests"
 	alertingClient "github.com/percona/pmm/api/managementpb/alerting/json/client"
 	"github.com/percona/pmm/api/managementpb/alerting/json/client/alerting"
+	"github.com/percona/pmm/managed/services/grafana"
 )
 
 // Note: Even though the IA services check for alerting enabled or disabled before returning results
 // we don't enable or disable IA explicit in our tests since it is enabled by default through
 // ENABLE_ALERTING env var.
```

Comment:
I understand ENABLE_ALERTING is no longer relevant, but it's counterpart DISABLE_ALERTING is.

Also, there are mentions of IA up above, which, AFAICS, get changed to Percona Alerting in this PR.

Reply:
Cleaned up this, and all `IA`, `ia`, `integrated alerting` entries.